supply-chain-security/compromises/README.md

```diff
@@ -8,16 +8,26 @@ can better understand the patterns and develop best practices and tools.
 
 We welcome additions to this catalog by 
 [filing an issue](https://github.com/cncf/sig-security/issues/new/choose) or
-[github pull request](https://github.com/cncf/sig-security)
+submitting a [pull request](https://github.com/cncf/sig-security) to our repo.
+
+Additions aren't intended to be a statement on the security stance and/or
+practices of any given project, or of open source as a whole. If anything,
+many of the stories showcase rapid response, evolving security practices in
+the community and an open and transparent approach to disclosure.
```

Comment:
I like this banner, personally. I just wanted to add a note that I acknowledge this change and approve it. I'd like to know @lumjjb's, @TheFoxAtWork's and @ultrasaurus's opinion on it.

Reply:
Love this banner.  Perhaps some rephrasing to be more clear:

> Contents of this repo and proposed additions are not a statement or opinion on the security stance and/or practices of a given project, of open source, or the community.  These articles and stories annotate the communities dedication to rapid response, evolving security practices, transparent disclosure, and enforcement of one of open sources founding principles, "[Linus's Law](https://en.wikipedia.org/wiki/Linus%27s_law)".

Reply:
Also perhaps adding:

> When submitting an addition, please review the [definitions](https://github.com/cncf/sig-security/blob/master/supply-chain-security/compromises/compromise-definitions.md) page to ensure the Type of Compromise on the details of the incidents as well as the Catalog itself are consistent.  If a definition doesn't exist or a new type of compromise needs added, please include that as well.